src/types/FilterType.ts

```diff
@@ -0,0 +1,5 @@
+export const enum FilterType {
+  All,
+  Completed,
+  Active,
+}
```

Comment:
[Optional] In my opinion it is better not to use numerical enums 
```suggestion
export const enum FilterType {
  All = "All",
  Completed = "Completed",
  Active = "Active",
}
```

Reply:
agree
If you do like that - you can make a map in component with links